pkg/config/config.go

```diff
@@ -105,7 +105,7 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 // messages.
 func (c *Config) LogDeprecations(l log.Logger) {
 	for _, d := range c.Deprecations {
-		level.Warn(l).Log("msg", fmt.Sprintf("DEPRECATION NOTICE: %s", d))
+		_ = level.Warn(l).Log("msg", fmt.Sprintf("DEPRECATION NOTICE: %s", d))
```

Comment:
You shouldn't have to add the `_` here FWIW:
https://github.com/grafana/agent/blob/e6745e60df86b1848fa8341aedc17b61cc2f0950/.golangci.yml#L70-L74

Reply:
My linter kept complaining about it in Goland so I just added this.

Reply:
Can you disable the linting rule in Goland? I'd really rather not have to add `_ =` to every time we log :)